test/e2e/manifest_test.go

```diff
@@ -435,7 +435,12 @@ var _ = Describe("Podman manifest", func() {
 		Expect(output).To(ContainSubstring("Writing manifest to image destination"))
 		Expect(output).To(ContainSubstring("Storing signatures"))
 
-		push = podmanTest.Podman([]string{"manifest", "push", "--tls-verify=false", "--creds=podmantest:wrongpasswd", "foo", "localhost:" + registry.Port + "/credstest"})
+		// Invalid compression format specified for gzip, it must fail
+		push = podmanTest.Podman([]string{"push", "--compression-format=gzip", "--compression-level=20", "--tls-verify=false", "--creds=" + registry.User + ":" + registry.Password, "--format=v2s2", "localhost:" + registry.Port + "/alpine:latest"})
```

Comment:
this is running `push` not `manifest push`
exit code checks are bad, you must check for the proper error message on stderr.